common/include/pcl/common/vector_average.h

```diff
@@ -105,8 +106,8 @@ namespace pcl
      protected:
         //-----METHODS-----
         //-----VARIABLES-----
-        unsigned int noOfSamples_;
-        real accumulatedWeight_;
+        unsigned int noOfSamples_ = 0;
+        real accumulatedWeight_ = 0;
         Eigen::Matrix<real, dimension, 1> mean_;
```

Comment:
We can default these Eigen Matrices to `Identity` too, right? (all all Eigen Matricec otherplaces)